packages/now-cli/src/util/dev/server.ts

```diff
@@ -741,6 +746,26 @@ export default class DevServer {
     return merged;
   }
 
+  populateVercelEnvVars(env: Env | undefined): Env {
+    if (!env) {
+      return {};
+    }
+
+    for (const name of Object.keys(env)) {
+      if (name === 'VERCEL_URL') {
+        const host = new URL(this.address).host;
+        env['VERCEL_URL'] = host;
+      } else if (name === 'VERCEL_REGION') {
+        env['VERCEL_REGION'] = 'dev1';
```

Comment:
@TooTallNate Do we want to use `dev` instead of `dev1` like leo suggested in [ch4611](https://app.clubhouse.io/vercel/story/4611)? Once we ship this, I don't think we can change it.

Reply:
I mean I explained my opinion about it in the ch story, but ultimately I don't think it's my call. If we do change it to `dev` we'll need to update [the documentation](https://vercel.com/docs/v2/edge-network/regions#routing) as well.